app/src/main/java/com/weatherxm/ui/photoverification/gallery/PhotoGalleryActivity.kt

```diff
@@ -92,7 +99,21 @@ class PhotoGalleryActivity : BaseActivity() {
         registerForActivityResult(
             ActivityResultContracts.PickVisualMedia()
         ) { uri ->
-            // TODO: track event
+            analytics.trackEventUserAction(
```

Comment:
Now that I see it again, this should be below after the `uri?.let {` to be 100% correct that the user selected a photo correctly.